homeassistant/helpers/selector.py

```diff
@@ -564,6 +565,40 @@ def __call__(self, data: Any) -> str:
         return agent
 
 
+class CountrySelectorConfig(TypedDict, total=False):
+    """Class to represent a country selector config."""
+
+    countries: list[str]
+    no_sort: bool
```

Comment:
select selector has a `sort` can we do that here too, and let it default to `True`?

Reply:
Reason for using `no_sort´ is to "mimic" the `LanguageSelector` e.g. used that as reference to build this.
I can of course reverse it to `sort` instead but not sure why not to leave it as is?

Reply:
OK, makes sense 👍